src/main/java/net/sf/jabref/event/ChangedFieldEvent.java

```diff
@@ -0,0 +1,37 @@
+package net.sf.jabref.event;
+
+import net.sf.jabref.model.entry.BibEntry;
+
+/**
+ * <code>ChangedFieldEvent</code> is fired when a field of <code>BibEntry</code> has been modified.
```

Comment:
Or removed. Or added. See `eventBus.post(new ChangedFieldEvent(this, fieldName, null));`. Please add this as comment.